python/setup.in.py

```diff
@@ -72,6 +72,7 @@ def get_tag(self):
     package_data= package_data_dict,
     cmdclass={'bdist_wheel': BinaryBdistWheel},
     extras_require={
+        'plaidml': ["tensorflow==1.13.0"],
```

Comment:
TODO check: now that master is at 1.14, this will not work?

Reply:
It should. Will find out :)